docs/configuration.md

```diff
@@ -111,6 +111,9 @@ The configuration file must be named `arduino-cli`, with the appropriate file ex
 - envfile
 - [INI]
 
+The configuration file [JSON schema][configuration-schema] can be used to independently validate the file content. This
```

Comment:
Contingent on my suggestion above being accepted, I think the best location for the "**JSON schema**" section is at the end of the parent "**Configuration file**" section:

- Configuration file
  - File name
  - Supported formats
  - Locations
  - Example
  - JSON schema

Reply:
Updated 👍